src/recipes/ceviche-with-shrimp.md

```diff
@@ -0,0 +1,42 @@
+---
+tags: [side dish]
+title: "Ceviche With Shrimp"
+---
+
+# Ceviche With Shrimp
+
+Ceviche is one of my favorite dishes that I enjoyed when I started my no red meat diet. I first tried Ceviche when my bestfriend brought it to work for Lunch. I loved it so much that I learned how to make it on my own.
+
+## Ingredients
+
+1 bag of cooked Shrimp
+1 Jalpeno Pepper
+1 Tomato
+1/2 purple onion
+1/2 bunch of cilantro
+6 limes
+Tortilla Chips
+
+
+## Tools
+
+Bowl
+juicer
+list of tools
+
+## Method
+
+Cut up the tomatoe, purple onion, jalpeno pepper, cilantro
+Add Ingredients to Bowl
+Peel shrimp tail off.
+Cut each lime in half and squeeze juice into bowl
+Mix all the ingredients together
+Let ceviche sit in fridge to chill
+
+## Tips
+
+To add a better taste, I also add lemon peper and garlic powder to add seasoning. you can add any kind of totilla chips to the dish.
```

Comment:
looks yummy! you might add this either to the dinner category or let it stay in 'everything else'